pkg/sensors/exec/procevents/proc_reader.go

```diff
@@ -417,11 +420,14 @@ func GetRunningProcs() error {
 
 		p := procs{
 			ppid: uint32(_ppid), pnspid: pnspid, pargs: pcmdsUTF,
-			pflags: api.EventProcFS | api.EventNeedsCWD | api.EventNeedsAUID,
-			pktime: pktime,
-			uid:    euid, // use euid to be compatible with ps
-			auid:   auid,
-			pid:    uint32(pid), nspid: nspid, args: cmdsUTF,
+			pflags:               api.EventProcFS | api.EventNeedsCWD | api.EventNeedsAUID,
+			pktime:               pktime,
+			uid:                  euid, // use euid to be compatible with ps
+			auid:                 auid,
+			pid:                  uint32(pid),
+			tid:                  uint32(pid), // Read dir does not return threads and we only track tgid
+			nspid:                nspid,
+			args:                 cmdsUTF,
```

Comment:
this seems to add more fields than just tid, and there's no changelog explanation for that

Reply:
Ah it just adds tid, the others were inlined and patch added new lines